### ISSUE TITLE
Version of requirements 

### ISSUE THREAD

**Issue by Qrrrr** [Vincent9797/Attention-Based-Two-Stream-Convolutional-Networks-for-Face-Spoofing-Detection#2]
Could you pleas inform the version of the requirements such as version of Tensorflow and Keras and other required things. Because I just cannot successfully run train.py, and I guess it may be the problem from my environment. Thx

**Comment by l347437585** [comment]
Have you solved this problem? I  tried several versions of Keras and Albumentations, and they are always missing functions.

**Comment by Qrrrr** [comment]
> Have you solved this problem? I tried several versions of Keras and Albumentations, and they are always missing functions.

Yes, I use Keras 2.3.1, tensorflow 2.2.0 and Albumentations 0.5.2. I forgot some details about how I make it run, because I train the model in win10 3 months ago, but I still remenber that there are some bugs in attentions.py and train.py (such that stream.name should be stream._name)

**Comment by Qrrrr** [comment]
> Have you solved this problem? I tried several versions of Keras and Albumentations, and they are always missing functions.

Yes, I use Keras 2.3.1, tensorflow 2.2.0 and Albumentations 0.5.2. I forgot some details about how I make it run, because I train the model in win10 3 months ago, but I still remenber that there are some bugs in attentions.py and train.py (such that stream.name should be stream._name)
